src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -135,11 +141,17 @@ public void killConnection(String key) {
    * Requests a token from the key server.
    */
   public String requestTokenFromServer() {
+    //TODO: Replace when the StunFileGeneration is ready
+    pathToStunDumpFile = this.getClass().getClassLoader().getResource("ice4jDemoDump.ice")
```

Comment:
I think it would be better to put this next to the initialization on line 36, since most probably we will be using a constant for the path anyways and overwrite the previous dump when new ones are generated I think